keyboards/dz60/keymaps/jackokring/AutoHotkey.ahk

```diff
@@ -0,0 +1,15 @@
+; A script to intercept Linux Unicode hotkey on Windows.
```

Comment:
This file needs to be removed. We want to keep the repo as clean as possible nowdays. I would suggest hosting in another repo, and maybe adding a link within one of the markdown files.

Reply:
ok